src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -62,23 +64,26 @@
      * Create a Post in to database.
      */
     @PostMapping(value = "/create")
-    public ResponseEntity<Object> createPost(@RequestBody PostDto post) {
-        PostEntity savedPost = postService.create(post);
-        SuccessResponse resp = new SuccessResponse(savedPost);
+    public ResponseEntity<?> createPost(@Validated @RequestBody PostDto post) {
+        PostDto newPostDto = postService.createPost(post);
+
+        SuccessResponse resp = new SuccessResponse(newPostDto);
```

Comment:
same thing return PostDto instead of entity will change the payload

Reply:
See above